src/app/inventory/dimCsvService.factory.ts

```diff
@@ -38,6 +42,21 @@ function downloadCsv(filename, csv) {
   document.body.removeChild(pom);
 }
 
+function buildSocketString(sockets: DimSockets): string {
+  const socketItems = sockets.sockets.map((s) =>
+    s.plugOptions
+      .filter((p) => FILTER_NODE_NAMES.every((n) => n !== p.plugItem.displayProperties.name))
```

Comment:
Better to filter on `!FILTER_NODE_NAMES.some((n) => n === p.plugItem.displayProperties.name)`